config.reek

```diff
@@ -0,0 +1,39 @@
+### Generic smell configuration
+
+# You can disable smells completely
+IrresponsibleModule:
+  enabled: false
+
+# You can use filters to silence Reek warnings.
+# Either because you simply disagree with Reek (we are not the police) or
+# because you want to fix this at a later point in time.
+# NestedIterators:
+#   exclude:
+#     - "MyWorker#self.class_method" # should be refactored
+#     - "AnotherWorker#instance_method" # should be refactored as well
+
+# A lot of smells allow fine tuning their configuration. You can look up all available options
+# in the corresponding smell documentation in /docs. In most cases you probably can just go
+# with the defaults as documented in defaults.reek.
+DataClump:
+  max_copies: 3
+  min_clump_size: 3
+
+### Directory specific configuration
+
+# You can configure smells on a per-directory base.
+# E.g. the classic Rails case: controllers smell of NestedIterators (see /docs/Nested-Iterators.md) and
+# helpers smell of UtilityFunction (see docs/Utility-Function.md)
+"web_app/app/controllers":
+  NestedIterators:
+    enabled: false
+"web_app/app/helpers":
+  UtilityFunction:
+    enabled: false
+
+### Excluding directories
+
+# Directories below will not be scanned at all
+exclude_paths:
```

Comment:
Looks like the whole file was copied from another project. I think so because you don't have `web_app/app/controllers` or `lib/legacy` folders. If it's true, you can start with something simpler:
```yml
IrresponsibleModule:
  enabled: false

NestedIterators:
  enabled: false
```
Then it might be adjusted on need.

Reply:
Well, it was found in some tutorial article/post. Thus probably yes :) Will change